packages/web/src/scss/components/Pagination/_Pagination.scss

```diff
@@ -0,0 +1,47 @@
+@use 'theme';
+@use '../../settings/cursors';
+
+.Pagination {
+    margin-inline: auto;
```

Comment:
I think this doesn't do anything?

Reply:
Refactored within changes across whole component.